app/models/equipment_model.rb

```diff
@@ -166,6 +166,14 @@ def document_attributes=(document_attributes)
     end
   end
 
+  def active_reservations
+    if AppConfig.first.requests_affect_availability
```

Comment:
great point; I didn't use it because we don't use it in most places. It would probably be good to switch most of these checks over to using the class method; should I open up an issue? It'd be a quick one